lnwallet/channel_test.go

```diff
@@ -7637,8 +7675,9 @@ func TestIdealCommitFeeRate(t *testing.T) {
 
 		// If the maximum fee is lower than the network fee and above
 		// the min relay fee, use the maximum fee.
+		maxFeeRate := maxFeeRate(aliceChannel, 1.0)
 		assertIdealFeeRate(
-			aliceChannel, 700000000, 0, 0, 1.0, 676794154,
+			aliceChannel, 700000000, 0, 0, 1.0, maxFeeRate,
```

Comment:
This test structure doesn't give me a lot of confidence overall. I understand that it precedes your changes here, but I think it's too fragile. Not sure it's worth fixing here though. What I'd like to see instead is the emergent hierarchy of fee choices as one test, and the ratcheting property you implement here as a separate test altogether.

Reply:
I tried to reorganize the tests, without anticipating the ratcheting property. I think it is better to keep all the test possibilities in one place but PTAL.